src/GitHub.Api/Application/ApiClient.cs

```diff
@@ -219,66 +220,69 @@ public ApiClient(UriString hostUrl, IKeychain keychain, IGitHubClient githubClie
             }
         }
 
-        private async Task<IList<Organization>> GetOrganizationInternal()
+        private async Task GetOrganizationInternal(Action<IList<Organization>> onSuccess, Action<Exception> onError = null)
```

Comment:
Related to https://github.com/github-for-unity/Unity/pull/300#discussion_r146317238, `IList<Organization>` is not a very useful type, especially since it won't be serializable by Unity (meaning we need to get a `ToArray()` here to get it and then we need to do a `ToList()` in the UI to serialize it, so copying the data twice for no big gains)

Also, `Organization` is not easily serializable and has too much junk in it that we don't need, we should wrap this in an object we control and not expose Octokit models directly to the outside.

Reply:
I opened up #386 to resolve this issue